user-service/README.md

```diff
@@ -100,6 +107,24 @@ curl -X POST user-service/api/user -H "x-request-token: test-request-token" -H "
 }'
 ```
 
+## User pre-population / Cache Warming
+
+This service can be populated with a list of initial users at runtime.
+By specifying all the configuration and pre-population factory values, any user information in the configuration list will be added to the service at runtime.
+This includes being added to the cache, so that the basic `NullUserService` implementation, coupled with the `UserServiceCachingProxy`, acts solely using the cache.
+
+**Note:** This means that after the cache eviction TTL (Time To Live), found in [application.yaml](src/main/resources/application.yaml), the User Service will no longer contain any user information.
+ 
+An example of this can be seen in this [Test Yaml](src/contract-tests/resources/application-pre-population.yaml) file which adds a basic user to the backing store when the service starts up.
```

Comment:
```suggestion
An example of this can be seen in this [Test Yaml](src/contract-tests/resources/application-pre-population.yaml) file which adds a basic user to the cache when the service starts up.
```

Reply:
Updated